site/pages/badger-profile/index.js

```diff
@@ -27,7 +27,7 @@ const BadgerProfile = ({ badger }: { badger: Badger }) => {
       <div className={styles.profileContainer}>
         <div className={styles.profilePictureContainer}>
           <img
-            src={badger.imageUrl}
+            src={badger.secondaryImageUrl || badger.primaryImageUrl}
```

Comment:
We might use a placeholder for that but I think falling back to the primary image is slightly better. We can ask designers for final decision.